opensearch/src/main/java/org/opensearch/sql/opensearch/request/OpenSearchRequestBuilder.java

```diff
@@ -173,14 +174,21 @@ public void pushDownLimit(Integer limit, Integer offset) {
    * Add highlight to DSL requests.
    * @param field name of the field to highlight
    */
-  public void pushDownHighlight(String field) {
+  public void pushDownHighlight(String field, Map<String, Literal> arguments) {
     if (sourceBuilder.highlighter() != null) {
       sourceBuilder.highlighter().field(StringUtils.unquoteText(field));
     } else {
       HighlightBuilder highlightBuilder =
           new HighlightBuilder().field(StringUtils.unquoteText(field));
       sourceBuilder.highlighter(highlightBuilder);
     }
+
+    if (arguments.containsKey("pre_tags")) {
+      sourceBuilder.highlighter().preTags(arguments.get("pre_tags").toString());
```

Comment:
What happens when multiple `highlight` calls specify different parameters? Last one wins?

Say, in a statement like `SELECT highlight(Tag, ..., pre_tags='<b>',...) , highlight(Title, ..., pre_tags='<i>') FROM ... `
will both Tag and Title highlights use `<i>` tag?

Reply:
Good catch on the bug!

Reply:
Resolved.